packages/deployment/app/data_storage.config.yml

```diff
@@ -0,0 +1,5 @@
+default:
+
+  # prevent this stack from referencing the step function config
+  # of the parent stack
+  stepFunctions: null
```

Comment:
This should be fine, however I'm believe that if you simply *have* the file, e.g. default:  it should accomplish that.